rs/http_endpoints/public/src/lib.rs

```diff
@@ -1126,8 +1135,9 @@ mod tests {
         let http_handler = HttpHandler {
             call_router: Router::new()
                 .route(call::CallServiceV2::route(), axum::routing::post(dummy)),
-            call_v3_router: Router::new()
-                .route(call::CallServiceV3::route(), axum::routing::post(dummy)),
+            call_v3_router: Some(
```

Comment:
I am wondering instead of having an optional router here. Can you just route /v3/ to the service of v2 ? I assume that axum processes the handlers in order.

@tthebst maybe can confirm this